examples/compose.js

```diff
@@ -0,0 +1,29 @@
+import dotty from 'dotty';
+import merge from 'lodash/object/merge';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return composables.map(composable => {
+    return dotty.get(composable, 'compose.' + descriptorName);
+  });
+};
+
+function compose (...composables) {
+  const composable = {};
```

Comment:
I thought `composable` is a [factory **function**](https://github.com/stampit-org/stamp-specification#composable). :)